adapter/pkg/messaging/listener.go

```diff
@@ -51,6 +63,22 @@ func StartConsumer(key string) (*Consumer) {
 	go func() {
 		c.reconnect(key)
 	}()
+	err := handleEvent(c, key)
+	if err != nil {
+		logger.LoggerMsg.Fatalf("%s", err)
+	}
+	if lifetime > 0 {
```

Comment:
It seems like lifetime is always zero. Can we please check this
https://github.com/wso2/product-microgateway/blob/1cde30dec45b60c3b9ac6b919b639e8b63260086/adapter/pkg/messaging/listener.go#L38

Reply:
We can either remove this or make it configurable.